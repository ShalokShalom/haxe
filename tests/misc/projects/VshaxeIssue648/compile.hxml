--main Main
--interp
-D message.reporting=pretty
-D message.no-color
-D nofail
