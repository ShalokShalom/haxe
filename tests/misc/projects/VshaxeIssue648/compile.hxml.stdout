Main.hx:5: Jeremy test
Main.hx:6: Jérémy test
Main.hx:7: 名 字 test
Main.hx:8: zя���� test abcdefghijk
Main.hx:9: ���� test abcdefghijk
Main.hx:10: zя test abcdefghijk
Main.hx:11: 😀 😀 test abcdefghijk
Main.hx:12: 😀 😀 zя���� test abcdefghijk
